.github/workflows/pre-commit.yml

```diff
@@ -16,5 +16,5 @@ jobs:
       with:
         # Note: this should match Cloud Composer
         # https://cloud.google.com/composer/docs/concepts/versioning/composer-versions
-        python-version: '3.8'
+        python-version: '3.10'
```

Comment:
Did the cloud composer version change as well?

Reply:
Thanks for asking! Yes, the composer env has been upgraded to 3.10 in https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/134